types/validtransaction_bench_test.go

```diff
@@ -0,0 +1,59 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/NebulousLabs/Sia/crypto"
+	"github.com/NebulousLabs/Sia/encoding"
+)
+
+// BenchmarkStandaloneValid times how long it takes to verify a single
+// large transaction, with 100 signatures
+func BenchmarkStandaloneValid(b *testing.B) {
+	// make a transaction 10 with valid inputs with valid signatures
+	b.ReportAllocs()
+	txn := Transaction{}
+	numSigs := 100
```

Comment:
I don't think that 100 signatures is a super interesting number. The majority of transactions are going to have somewhere between 1 and 10 signatures, so the benchmark should be in that range.

The other interesting number is the really large transactions, because people can create those to be malicious. But I don't think there's a standard use case for 100 signature transactions - it'll neither apply to an attack case (b/c more signatures is more powerful) nor a standard case.